src/Handshake/RequestVerifier.php

```diff
@@ -113,42 +111,42 @@ function ($x) {
      * @todo The spec says we don't need to base64_decode - can I just check if the length is 24 and not decode?
      * @todo Check the spec to see what the encoding of the key could be
      */
-    public function verifyKey(array $keyHeader) {
-        return (1 === count($keyHeader) && 16 === strlen(base64_decode($keyHeader[0])));
+    public function verifyKey(array $keyHeader): bool {
+        return 1 === count($keyHeader) && 16 === strlen(base64_decode($keyHeader[0]));
     }
 
     /**
      * Verify the version passed matches this RFC
      * @param string[] $versionHeader MUST equal ["13"]
      * @return bool
      */
-    public function verifyVersion(array $versionHeader) {
-        return (1 === count($versionHeader) && static::VERSION === (int)$versionHeader[0]);
+    public function verifyVersion(array $versionHeader): bool {
+        return 1 === count($versionHeader) && static::VERSION === (int)$versionHeader[0];
     }
 
     /**
      * @todo Write logic for this method.  See section 4.2.1.8
      */
-    public function verifyProtocol($val) {
+    public function verifyProtocol($val): bool {
```

Comment:
Assuming this and `verifyExtensions` will also have boolean returns when implemented.  So, adding them in now to avoid the theoretical B/C break later by changing the return type from void to bool.